internal/wrappers/utils/utils_test.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"log"
+	"testing"
+)
+
+func TestCleanURL(t *testing.T) {
```

Comment:
@igorlombacx  Can you please split it into different methods instead of having the for loop ?

Reply:
yes : )